.github/integration/scripts/charts/deploy_charts.sh

```diff
@@ -39,11 +39,18 @@ if [ "$1" == "sda-mq" ]; then
 fi
 
 if [ "$1" == "sda-svc" ]; then
+    inbox=s3
+    if [ "$3" == "true" ] && [ "$4" == "posix" ]; then
```

Comment:
Why do we ignore inbox type if not using tls?

Reply:
The sftp inbox that uses posix as backend needs TLS.

Reply:
How about excluding the bad case (tls=false, storage=posix) instead (https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations) of testing something other than what one would assume?